datafusion/proto/src/logical_plan/from_proto.rs

```diff
@@ -1342,6 +1347,10 @@ pub fn parse_expr(
                     parse_expr(&args[0], registry)?,
                     parse_expr(&args[1], registry)?,
                 )),
+                ScalarFunction::ArrayIntersect => Ok(array_intersect(
```

Comment:
I dont why without this can pass CI.

Reply:
Do we need to enhance the check during CI? maybe this is a little bug?